gnocchi/cli/metricd.py

```diff
@@ -240,6 +247,9 @@ def _run_job(self):
                 s_count += 1
                 self.incoming.finish_sack_processing(s)
                 self.sacks_with_measures_to_process.discard(s)
+            except ConnectionError as ce:
+                LOG.debug("Redis server closed connection. Retrying.")
+                raise tenacity.TryAgain(ce)
```

Comment:
The function is not directly decorated with tenacity so I'd avoid raising this exception here.
Maybe it'd be better to decorate this with a tenacity decorator?

Reply:
done